site/src/components/events/filter/useEventSectionMethodFilter.js

```diff
@@ -90,6 +91,10 @@ export function useEventSectionMethodFilter() {
         currentFilterValue.method ?? getFromQuery(location, "method");
       const sectionValue =
         currentFilterValue.section ?? getFromQuery(location, "section");
+      const extrinsicOnlyValue =
+        currentFilterValue.is_extrinsic ?? getFromQuery(location, "is_extrinsic", "true");
+      //const noExtrinsicValue =
```

Comment:
Can you please remove commented out code so we don't grow dead code in the project over time? Thanks!

Reply:
I left that in on purpose because that filter is also one of the default ones that is missing but in my local testing, changing the filter value didn't do anything so I decided to keep it in but comment it for now, just in case it turns out it is needed in the future. But I can remove the comments no problem https://github.com/3Dpass/explorer/blob/932196d5e43dc4b7792db876ba38678ecf5d736e/site/src/pages/events.js#L26-L51

Reply:
I see, thank you!